src/Control/Applicative/Compat.hs

```diff
@@ -1,3 +1,6 @@
+#if !MIN_VERSION_base(4,8,0)
+{-# LANGUAGE StandaloneDeriving #-}
+#endif
```

Comment:
Again, if we can remove the CPP conditional here without adverse effects, I would vote for that.